ssl_cert_tracker/tests/test_models.py

```diff
@@ -15,19 +15,21 @@
 from datetime import datetime
 
 from django.test import TestCase
-from ssl_cert_tracker.models import NmapCertsData
+from ssl_cert_tracker.models import NmapCertsData, NmapHistory
 
-class SslCertTrackerTestModels(TestCase):
-    """Ssl_Cert_Tracker_Test_Models"""
+class TestSslCertTrackerModels(TestCase):
+    """TestSslCertTrackerModels injects different sets of values
+       into model fields and verifies the result
+    """
     @classmethod
     def setUpTestData(cls):
-        """setUpTestData"""
+        """setUpTestData: Initializes all objects/attributes"""
         cls.vNot_Before = None
         cls.vNot_After = None
         cls.obj = None
 
     def setUp(self):
-        """setUp"""
+        """setUp test data"""
         NmapCertsData.created_by = NmapCertsData.get_or_create_user(username='PHSA_User')

```

Comment:
hard coding usernames is a bad idea. make this an entry in settings and read the value from there